app/mailers/sac_cas/application_mailer.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module SacCas::ApplicationMailer
+  private
+
+  def compose(recipients, content_key, locales = [])
+    return if recipients.blank?
+
+    values = values_for_placeholders(content_key)
+    custom_content_mail(recipients, content_key, values, {}, locales)
+  end
```

Comment:
This doesn't seem covered by the specs. Is it needed?

Reply:
it has a new `locales` attribute, that's why

```rb
def compose(recipients, content_key, locales = [])
  return if recipients.blank?

  values = values_for_placeholders(content_key)
  custom_content_mail(recipients, content_key, values, {}, locales)
end
```

Reply:
it has a new `locales` attribute, that's why